src/analysis/complex/isometry.lean

```diff
@@ -45,6 +45,13 @@ def rotation : circle →* (ℂ ≃ₗᵢ[ℝ] ℂ) :=
 @[simp] lemma rotation_symm (a : circle) : (rotation a).symm = rotation a⁻¹ :=
 linear_isometry_equiv.ext $ λ x, rfl
 
+@[simp] lemma rotation_trans (a b : circle) :
+  (rotation a).trans (rotation b) = rotation (b * a) :=
+begin
+  ext1 x,
+  simp
+end
```

Comment:
We often compress super-short proofs like this into a single line, for example like this:
```suggestion
by { ext1, simp }
```